src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -639,3 +641,52 @@ ErrorIfAnyMetadataNodeOutOfSync(List *metadataNodeList)
 		}
 	}
 }
+
+
+/*
+ * IsWorkerTheCurrentNode checks if the given worker refers to the
+ * the current node by comparing the server id of the worker and of the
+ * current nodefrom pg_dist_node_metadata
+ */
+bool
+IsWorkerTheCurrentNode(WorkerNode *workerNode)
```

Comment:
Not sure if the usage of `worker` is good as we could be dealing with the coordinator. Though, I couldn't come up with a better alternative, if you can, please update the name

Reply:
True, but the parameter of this is a worker node, and if you look at the code logic, it tries to check if the worker node is the node executing the code. 
I could change it to: `IsTargetNodeTheCurrentNode`, but I think the current name is reflective of what the function does.